site/content/concepts/command-and-control.md

```diff
@@ -66,3 +66,24 @@ The response of the command will be sent by the device to
 If the device is authenticated, the `tenant` and `device-id` are left empty (resulting in 3 subsequent `/`s).
 
 ![Command & Control over MQTT Adapter](../command_control_concept_mqtt.png) 
+
+## Command & Control over AMQP Adapter
+
+When an authenticated device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link at the source address:
+
+* `control`
+
+while unauthenticated devices have to fully specify their `tenant` and `device-id` as source address. 
+
+Once the link is opened, the device can receive command messages from the adapter. The AMQP adapter sets the
+`reply-to` and `subject` properties of the message as follows:
+
+* `reply-to`: `control///req/<req-id>`(authenticated device) or `control/tenant/device-id/<req-id>` (unauthenticated device). The `<req-id>` can be used by the device when sending the command response to the adapter.
+* `subject` Contains the actual `<command>` to be executed by the device.
+
+Devices can publish command response messages (similar to how `telemetry` and `event` messages are published) using the following message address:
+
+* `control///res/<req-id>/<status>` (authenticated device)
+* `control/tenant/device-id/res/<req-id>/<status>` (unauthenticated device)
```

Comment:
IMHO this must be `control/tenant/device-id`.
Containing the other params as properties.

Reply:
ok :+1: